requirements.txt

```diff
@@ -2,5 +2,6 @@ pydantic==1.8.2
 requests==2.26.0
 pandas==1.3.0
 pyyaml==5.4.1
+fsspec==2021.08.1
```

Comment:
Is this used?

Reply:
This is not directly used, but an error while testing arose noting an outdated version of fsspec used by pandas.  However, after uninstalling and retesting, I could not reproduce the error.